### DISCUSSION TITLE
Snipe-IT SQL database map

### DISCUSSION THREAD

**Discussion by Jaspotn0612** [snipe/snipe-it#9620] [Q&A]
Hey all,

During a web search for info I came across this SQL map for Snipe-IT:
https://drawsql.app/templates/snipe-it

I have two questions:

- Is this a map done by the team at Snipe-it themselves or someone else's pet project? I can't see any names about who did it, though since it has 37 tables it was attached to a paid account at some point.

- It was last updated a year ago so have there been significant changes since then?

**Comment by snipe** [comment]
I'm pretty sure someone else created that - I don't remember doing it. There have definitely been some changes to the database over the past year or so.
